src/ludamus/links/db/django/repositories.py

```
@@ -148,6 +149,22 @@
+logger = logging.getLogger(__name__)
+
+
+def _delete_stored_file(field_file: object, old_name: str) -> None:
+    # Best-effort: a storage hiccup (e.g. GCS) must not fail an otherwise
+    # successful DB update, so log and swallow cleanup errors rather than
+    # letting them propagate. Broad by design — the backend can raise anything.
+    if (storage := getattr(field_file, "storage", None)) is None:
+        return
+    try:
+        storage.delete(old_name)
+    except Exception:  # pylint: disable=broad-exception-caught
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)